apps/web/src/components/views/CompanyInfoView/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "@dotkomonline/ui"
+import { FC } from "react"
+
+import PortableText, { PortableTextProps } from "@/components/molecules/PortableText"
+
+import CompanyInterestProcess from "./CompanyInterestProcess"
+import OurProducts from "./OurProducts"
+import { Title } from "@dotkomonline/ui"
+
+export type Content = { content: PortableTextProps["blocks"] }
+
+interface CompanyInfoViewProps {
+  companyInfoContent: Content[]
+}
+export const CompanyInfoView: FC<CompanyInfoViewProps> = (props: CompanyInfoViewProps) => {
+  const [header, interest, product, info, contact] = props.companyInfoContent
```

Comment:
Maybe make it an object instead? It's harder to keep track of an array index over time :p

Reply:
Resolving as its unrelated to this PR.